components/pages/aunthentication/auth.css

```diff
@@ -35,12 +37,18 @@
     margin: 1.5rem;
     width: 8rem;
     color: black;
+    cursor: pointer;
 }
 
 .sign-up{
     color:tomato;
```

Comment:
For colors, use hex code or any other ways than some defined name. And use variables to assign them.